test/norm_num.lean

```diff
@@ -62,25 +62,22 @@ example : (12321 - 2 : ℤ) = 12319 := by norm_num
 example (x : ℤ) (h : 1000 + 2000 < x) : 100 * 30 < x :=
 by norm_num at *; exact h
 
-example : (1103 : ℤ) ≤ (2102 : ℤ) := by norm_num
-example : (110474 : ℤ) ≤ (210485 : ℤ) := by norm_num
-example : (11047462383473829263 : ℤ) ≤ (21048574677772382462 : ℤ) := by norm_num
-example : (210485742382937847263 : ℤ) ≤ (1104857462382937847262 : ℤ) := by norm_num
-example : (210485987642382937847263 : ℕ) ≤ (11048512347462382937847262 : ℕ) := by norm_num
--- example : (210485987642382937847263 : ℚ) ≤ (11048512347462382937847262 : ℚ) := by norm_num
-example : (2 * 12868 + 25705) * 11621 ^ 2 ≤ 23235 ^ 2 * 12868 := by norm_num
+-- example : (1103 : ℤ) ≤ (2102 : ℤ) := by norm_num1
+-- example : (110474 : ℤ) ≤ (210485 : ℤ) := by norm_num1
+-- example : (11047462383473829263 : ℤ) ≤ (21048574677772382462 : ℤ) := by norm_num1
+-- example : (210485742382937847263 : ℤ) ≤ (1104857462382937847262 : ℤ) := by norm_num1
+-- example : (210485987642382937847263 : ℕ) ≤ (11048512347462382937847262 : ℕ) := by norm_num1
+-- example : (210485987642382937847263 : ℚ) ≤ (11048512347462382937847262 : ℚ) := by norm_num1
+-- example : (2 * 12868 + 25705) * 11621 ^ 2 ≤ 23235 ^ 2 * 12868 := by norm_num1
```

Comment:
Please uncomment.

Reply:
These don't actually work yet as-is, because norm_num doesn't handle `le` by itself (hence `norm_num1` fails). Previously they just read `by norm_num` and were actually being handled by simp, not norm_num. Should they still be uncommented?

Reply:
Ah, got it!  Then it's fine.